src/test/java/com/shippo/model/BatchTest.java

```diff
@@ -0,0 +1,132 @@
+package com.shippo.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.util.concurrent.TimeoutException;
+
+import org.junit.Test;
+
+import com.shippo.exception.APIConnectionException;
+import com.shippo.exception.APIException;
+import com.shippo.exception.AuthenticationException;
+import com.shippo.exception.InvalidRequestException;
+import com.shippo.model.Batch.BatchStatus;
+
+public class BatchTest extends ShippoTest {
+
+	final String id = "dd442663c24843068977704b1bd7d91d";
```

Comment:
Actually it may be better off like this since all methods in this class are instance methods. It's object is created by junit and call's methods on that object. I don't have very strong opinion about this. I can change it if you feel strongly about it.